chronograf.go

```diff
@@ -143,6 +187,13 @@ type DashboardQuery struct {
 	QueryConfig QueryConfig `json:"queryConfig,omitempty"` // QueryConfig represents the query state that is understood by the data explorer
 }
 
+// TemplateQuery is used to retrieve choices for template replacement
+type TemplateQuery struct {
+	Command string `json:"query"`        // Command is the query itself
+	DB      string `json:"db,omitempty"` // DB is optional and if empty will not be used.
+	RP      string `json:"rp,omitempty"` // RP is a retention policy and optional; if empty will not be used.
```

Comment:
It would also be useful for the front-end to be able to send and save the user's selection of measurement, fieldKey, TagKey, and TagValue